tools/yajsw/bin/installDaemon.sh

```diff
@@ -36,7 +36,7 @@ fi
 if [ -z "${JAVA_HOME}" ]; then
   echo -e "\nNo JAVA_HOME environment variable found!"
   echo "Attempting to determine JAVA_HOME (if this fails you must manually set it)..."
-  java_bin=$(readlink -f `which java`)
+  java_bin=$(readlink `which java`)
```

Comment:
Why was `-f` removed?

`-f` on Linux (GNU CoreUtils) is quite different to Mac (BSD). I cannot see an equivalent on Mac. I think this needs to be switched based on OS, so that we still use `-f` on Linux but on Mac we do whatever the equivalent is.